builtin/providers/vsphere/resource_vsphere_virtual_machine_test.go

```diff
@@ -33,7 +33,7 @@ func TestAccVSphereVirtualMachine_basic(t *testing.T) {
 	if v := os.Getenv("VSPHERE_DATASTORE"); v != "" {
 		datastoreOpt = fmt.Sprintf("        datastore = \"%s\"\n", v)
 	}
-	template := os.Getenv("VSPHERE_TEMPLATE")
+	cloneSource := os.Getenv("VSPHERE_CLONE_SOURCE")
```

Comment:
Yah ... we have a lot of code already.  Major refactor is going to be fun.  As you go along please make comments like ... Need to refactor this to here ... blah blah